openassessment/assessment/test/test_staff.py

```diff
@@ -117,6 +123,9 @@ def test_create_assessment_required(self, key):
         # Verify assesment made, score updated, and no longer waiting
         self.assertEqual(staff_assessment["points_earned"], OPTIONS_SELECTED_DICT[key]["expected_points"])
         self._verify_done_state(tim_sub["uuid"], self.STEP_REQUIREMENTS_WITH_STAFF)
+        # Verify that a StaffWorkflow step has been marked as complete
+        workflow = StaffWorkflow.objects.get(submission_uuid=tim_sub['uuid'])
```

Comment:
Now that we're on django 1.8, refreshing a previously-fetch object from the database is possible: https://docs.djangoproject.com/en/1.8/ref/models/instances/#refreshing-objects-from-database